tdrs-backend/tdpservice/parsers/constants.py

```diff
@@ -0,0 +1,169 @@
+"""Define constants for the parsers module."""
+
+TANF_ITEM_NAME = {
+    "AID_AGED_BLIND": "Aid to the Aged, Blind, and Disabled",
+    "AMOUNT_EARNED_INCOME": "Amount of Earned Income",
+    "AMOUNT_UNEARNED_INCOME": "Amount of Unearned Income",
+    "AMT_FOOD_STAMP": "Amount of Food Stamps",
+    "AMT_SUB_CC": "Amount of Subsidized Child Care",
+    "CC_NBR_MONTHS": "Number of Months of Subsidized Child Care",
+    "CHILD_SUPPORT_AMT": "Amount of Child Support",
+    "COMM_SERVICES_EA": "Community Service - excused absence",
+    "COMM_SERVICES_HOL": "Community Service - holiday",
+    "COMM_SERVICES_HOP": "Community Service - hours of participation",
+    "COOPERATION_CHILD_SUPPORT": "Cooperation with Child Support",
+    "COUNTABLE_MONTHS_STATE": "Countable Months of State Time",
+    "COUNTABLE_MONTH_FED": "Countable Months of Federal Time",
+    "COUNTY_FIPS_CODE": "County FIPS Code",
+    "CURRENT_MONTH_STATE": "Current Month of State Time",
+    "DATE_OF_BIRTH": "Date of Birth",
+    "DEEMED_HOURS_FOR": "Deemed Hours for",
+    "DISABLED_TITLE_XIVAPDT": "Disabled under title XIV-APDT",
+    "ED_NO_HIGH": "Education - No High School Diploma",
+    "FAILURE_TO_COMPLY": "Failure to Comply",
+    "FAMILY_CASH_RESOURCES": "Family Cash Resources",
+    "FAMILY_EXEMPT_TIME": "Family Exempt Time",
+    "FAMILY_NEW_CHILD": "Family with a New Child",
+    "FAMILY_SANC_ADULT": "Family Sanctioned for an Adult",
+    "FED_DISABILITY_STATUS": "Federal Disability Status",
+    "FED_OASDI_PROGRAM": "Federal OASDI Program",
+    "JOB_SEARCH_EA": "Job Search - EA",
+    "JOB_SEARCH_HOL": "Job Search - HOL",
+    "JOB_SEARCH_HOP": "Job Search - HOP",
+    "JOB_SKILLS_TRAINING": "Job Skills Training",
+    "MONTHS_FED_TIME": "Months of Federal Time",
+    "MONTHS_STATE_TIME": "Months of State Time",
+    "NBR_FAMILY_MEMBERS": "Number of Family Members",
+    "NEEDS_OF_PREGNANT": "",
+    "NEEDS_PREGNANT_WOMAN": "Needs",
```

Comment:
Oh, is this supposed to be the same as line 37?

Reply:
I have just grabbed all the item names, I guess we could change it to match the other one.